### DISCUSSION TITLE
Idea: Compact reference documentation of the javascript API

### DISCUSSION THREAD

**Discussion by kzvi** [johnlindquist/kit#861] [Ideas]
The "docs" tab in the app is nice for learning, but I think it could be more useful for reference to have a terse list of the javascript functions that are provided to scripts somewhere. Like a programming language standard library documentation, for example.

**Comment by johnlindquist** [comment]
Yeah, a reference is definitely very high on my list.

I started working on docs/examples here as well:

https://www.scriptkit.com/docs

I'd like to copy the Tailwind/Vue docs approach. It's just a matter of making time for it. I've got a BIG release coming out this weekend :D
